supervisor.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+set -e
+
+trap 'kill %1; kill %2' ERR EXIT
+
+cmd=./transfer2go
```

Comment:
Rishi, could you please allow to read name of the executable. In deployment procedure we should not rely on relative path and the fact that both will be in the same directory. Therefore I suggest:
```
cmd=$1
args=${2+"$@"}
```